src/dsp_tools/commands/xmlupload/stash/upload_stashed_resptr_props.py

```diff
@@ -6,6 +6,7 @@
 
```

Comment:
Our PR titles are used to create the release notes, so we should try to convey helpful information about what was fixed. For our users, it doesn't matter that we had a problem with a shallow copy. Rather, they want to know what kind of problem was solved in which dsp-tools command. So I suggest:

### fix(xmlupload): don't omit any stashed links (DEV-3827)

Reply:
Makes sens, I will change the title